improver_tests/acceptance/test_weighted_blending.py

```diff
@@ -134,17 +134,54 @@ def test_percentile(tmp_path):
     acc.compare(output_path, kgo_path)
 
 
-@pytest.mark.slow
-def test_cycletime(tmp_path):
-    """Test cycletime blending"""
+def test_cycletime_use_latest_frt(tmp_path):
+    """Test cycletime blending where the output is expected to have time
+    coordinates that match the latest of the input cubes."""
     kgo_dir = acc.kgo_root() / "weighted_blending/cycletime"
     kgo_path = kgo_dir / "kgo.nc"
-    input_path = kgo_dir / "input.nc"
+    input_paths = sorted((kgo_dir.glob("input_temperature*.nc")))
+    output_path = tmp_path / "output.nc"
+    args = ["--coordinate", "forecast_reference_time",
+            "--y0val", "1.0",
+            "--ynval", "4.0",
+            *input_paths,
+            "--output", output_path]
+    run_cli(args)
+    acc.compare(output_path, kgo_path)
+
+
+def test_cycletime_with_specified_frt(tmp_path):
+    """Test cycletime blending where a forecast reference time for the
+    returned cube is user specified."""
+    kgo_dir = acc.kgo_root() / "weighted_blending/cycletime"
+    kgo_path = kgo_dir / "kgo_specified_frt.nc"
+    input_paths = sorted((kgo_dir.glob("input_temperature*.nc")))
+    input_paths.pop(-1)
+    output_path = tmp_path / "output.nc"
+    args = ["--coordinate", "forecast_reference_time",
+            "--y0val", "1.0",
+            "--ynval", "4.0",
+            "--cycletime", "20200218T0500Z",
```

Comment:
Did you mean this to be T0500Z or T0600Z?

Reply:
Not sure how I managed that. 0600Z is what I was after. Altered and KGO updated.